### ISSUE TITLE
upgrade libvirt

### ISSUE THREAD

**Issue by prezha** [kubernetes/minikube#9642]
we are using libvirt v3.4.0 (from june 2017) and the current one atm is v6.9.0 (from nov 2020)

list of improvements made in the meantime is quite big: https://libvirt.org/news.html

**Comment by afbjorklund** [comment]
See also: https://packages.ubuntu.com/libvirt-daemon and https://packages.ubuntu.com/libvirt-bin

If you upgrade the minimum version requirement, that will exclude some of the older releases ?

Ubuntu | Libvirt
--- | ---
xenial (16.04LTS) | 1.3.1
bionic (18.04LTS) | 4.0.0
focal (20.04LTS) | 6.0.0

This for the linked ABI

**Comment by prezha** [comment]
when we revisit this, freshly released v7.1.0 brings (https://libvirt.org/news.html#v7-1-0-2021-03-01):
```
Implement Apple Silicon support
libvirt now runs on the ARM-based Apple Silicon Macs.
```
